api/views.py

```diff
@@ -305,3 +305,63 @@ def get_serializer_class(self):
         if self.request.method not in permissions.SAFE_METHODS:
             return CostumeSerializerForEditing
         return CostumeSerializer
+
+############################################################
+# Item
+
+class ItemSerializer(MagiSerializer):
+    image = ImageField()
+    i_type = IField(models.Item, 'type', required=False)
+
+    class Meta:
+        model = models.Item
+        fields = ('id', 'name', 'image', 'i_type', 'm_description')
+
+class ItemViewSet(viewsets.ModelViewSet):
+    queryset = models.Item.objects.all()
+    serializer_class = ItemSerializer
+    permission_classes = (api_permissions.IsStaffOrReadOnly, )
+
+############################################################
+# AreaItem
+
+class AreaItemSerializer(MagiSerializer):
+    image = ImageField()
+    i_type = IField(models.AreaItem, 'type', required=False)
+    i_instrument = IField(models.AreaItem, 'instrument', required=False)
+    i_attribute = IField(models.AreaItem, 'attribute', required=False)
+    i_stat = IField(models.AreaItem, 'stat', required=False)
+
+    class Meta:
+        model = models.AreaItem
+        fields = ('id', 'name', 'image', 'area', 'i_type', 'i_instrument', 'member', 'i_attribute', 'i_stat', 'values', 'lifes', 'about')
```

Comment:
It's probably a good idea to export the `values` and `lifes` fields in a more usable format. I think the right way to do that is to create \@properties for sanitized versions in the model, then have a separate serializer for editing the raw fields.

The Card API does this for skill variables, so you can look at that for reference. Or ask @db0company ...